scripts/zones/Mount_Zhayolm/mobs/Cerberus.lua

```diff
@@ -37,5 +37,5 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    mob:setRespawnTime((math.random(0,24)*3600)+172800); -- 48-72 hours proper 1 hour windows
+    mob:setRespawnTime(math.random(172800,259200)); -- 48-72 hours proper 1 hour windows
```

Comment:
Hmm, bleh, choices...
1. Change this one back, and leave them mismatched (Well, this and the zone)
2. To see if anyone wants to merge this as consistent
3. To change all others so they are consistent with windowing principles...

Meh, I guess if we wish to go with more authentic, I can window all mob:setRespawnTime instances. Fortunately, there are just a bit over 50, so it would be a little timely, but not overly.

Preference?